src/uiSelectSortDirective.js

```diff
@@ -92,6 +92,8 @@ uis.directive('uiSelectSort', ['$timeout', 'uiSelectConfig', 'uiSelectMinErr', f
         }
 
         move.apply(theList, [droppedItemIndex, newIndex]);
+        
+        scope.$parent.$selectMultiple.updateModel();
```

Comment:
Instead of going through `$parent` can't we `require: ['^uiSelect', '?^uiSelectMultiple']` and access via a `ctrls[]` param?